### DISCUSSION TITLE
Helper Codeigniter Yang Berguna - Tulisan Programmer Gadungan

### DISCUSSION THREAD

**Discussion by giscus** [topidesta/topidesta#24] [Comments]
# Helper Codeigniter Yang Berguna - Tulisan Programmer Gadungan

Bismillah .... Kebetulan development internal masih menggunakan PHP 5, dan beberapa fungsi ada yang gak berjalan, langkah terbaiknya dengan…

https://topidesta.my.id/helper-codeigniter-yang-berguna/

**Comment by mdestafadilah** [comment]
Gak dilanjut gan?

**Reply by mdestafadilah** [reply]
bodo amat
